nbs/13_questionnaire_handler.ipynb

```diff
@@ -33,20 +41,54 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 5,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "<jemalloc>: MADV_DONTNEED does not work (memset will be used instead)\n",
```

Comment:
please fix these prints because its shown in our documentation automatically

Reply:
check my enviroment

Reply:
maybe we can chat about this - trying to find a solution - can't find much online

Reply:
What are your pandas and numpy versions?

Reply:
2.1.0 pandas and 1.26.0 numpy